src/System.Data.SqlClient/src/Resources/Strings.resx

```diff
@@ -1042,4 +1042,7 @@
   <data name="LocalDBNotSupported" xml:space="preserve">
     <value>LocalDB is not supported on this Platform.</value>
   </data>
+  <data name="PlatformNotSupported_DataSqlClient" xml:space="preserve">
+    <value>System.Data.SqlClient is not supported on this platform.</value>
+  </data>
```

Comment:
Folks will hit it when targeting win8.1 or wpa8.1.  Both of those support netstandard1.2 and don't use RID for restore (packages.config).